proto/tendermint/abci/types.proto

```diff
@@ -157,11 +165,15 @@ message RequestVerifyVoteExtension {
 }
 
 message RequestFinalizeBlock {
-  bytes                   hash                 = 1;
-  tendermint.types.Header header               = 2 [(gogoproto.nullable) = false];
-  repeated bytes          txs                  = 3;
-  CommitInfo              decided_last_commit  = 4 [(gogoproto.nullable) = false];
-  repeated Evidence       byzantine_validators = 5 [(gogoproto.nullable) = false];
+  repeated bytes            txs                  = 1;
+  CommitInfo                decided_last_commit  = 2 [(gogoproto.nullable) = false];
+  repeated Evidence         byzantine_validators = 3 [(gogoproto.nullable) = false];
```

Comment:
this is more of a nit but I'd prefer if this were: 
```suggestion
  repeated Misbehavior         byzantine_validators = 3 [(gogoproto.nullable) = false];
```
So that we had a clearer line between types.Evidence (which can actually be proveable) and abci.Evidence which is just a list of validators and their crimes

Reply:
Changed.